integration-tests/src/test/add-ask-order.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Guid } from 'js-guid';
```

Comment:
It seems like you've added these new files but the old ones are still present. 

B/c they show up as new files we can't see what's the difference with the existing ones. Maybe consider `git mv` to rename them instead ?

Reply:
The changes are drastic enough to setup that I just created a new "project", so not sure this will be possible. I don't want to get rid of the old before I'm done with the new so not sure `git mv` will work.